flutter/news/lib/features/frontpage/data/datasource/articles_data_source.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:news/features/frontpage/data/models/article_model.dart';
+
+abstract class ArticlesRemoteDataSource {
+  Future<List<ArticleModel>> getTopHeadlines();
+  Future<List<ArticleModel>> getEverythingAbout(String query);
+}
```

Comment:
no newline at the end of file

Reply:
updated on https://github.com/novoda/multiplatform-playground/commit/8323a52c3c021a85d1662fa2609c061534a5fce3